src/App.js

```diff
@@ -12,60 +9,54 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
-      movieID: 0
-      // error: ''
+      movieID: 0,
+      error: ''
     }
   }
 
-    // componentDidMount() {
-    //   fetch('http://localhost:3001/api/v1/ideas')
-    //   .then(response => response.json())
-    //   .then(data=> {
-    //     this.setState({ideas: data})
-    //     console.log(data, "This is our data array")
-    //   })
-    //   .catch(() => this.setState({error: "Something went wrong!"}))
-    // }
-
+    componentDidMount() {
+      fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
```

Comment:
I'd take a moment to review the rubric - fetch calls are supposed to be defined in their own file.